.env.dev

```diff
@@ -0,0 +1,6 @@
+LLAMA_STACK_BASE_URL=http://localhost:5001
+LLAMA_STACK_CLIENT_LOG=debug
+LLAMA_STACK_PORT=5001
```

Comment:
I thought we removed this variable?

Reply:
Seems like we still use it https://github.com/search?q=repo%3Ameta-llama%2Fllama-stack%20LLAMA_STACK_PORT&type=code